eng/pipelines/templates/steps/archetype-sdk-docs.yml

```diff
@@ -0,0 +1,133 @@
+steps:
+  - pwsh: |
+      Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://github.com/mono/api-doc-tools/releases/download/mdoc-5.7.4.9/mdoc-5.7.4.9.zip" `
+      -OutFile "mdoc.zip" | Wait-Process; Expand-Archive -Path "mdoc.zip" -DestinationPath "./mdoc/"
+    workingDirectory: $(Build.BinariesDirectory)
+    displayName: Download and Extract mdoc Zip
+  - pwsh: |
+      Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://github.com/dotnet/docfx/releases/download/v2.43.2/docfx.zip" `
+      -OutFile "docfx.zip" | Wait-Process; Expand-Archive -Path "docfx.zip" -DestinationPath "./docfx/"
+    workingDirectory: $(Build.BinariesDirectory)
+    displayName: Download and Extract DocFX
+  - pwsh: |
+      dotnet restore "$(Build.SourcesDirectory)/eng/docgeneration/assets/docgen.csproj" /p:BuildBinariesDirectory=$(Build.BinariesDirectory)
+    displayName: Restore Nuget Packages Needed
+  - task: DotNetCoreInstaller@2
+    displayName: "Use .NET Core sdk $(DotNetCoreSDKVersion)"
+    inputs:
+      version: $(DotNetCoreSDKVersion)
+  - ${{ if ne(variables['Build.DefinitionName'], 'net - client - ci')}}:
+    - ${{ each artifact in parameters.Artifacts }}:
+      - pwsh: |
+          echo "##vso[task.setvariable variable=packageLocation]${{parameters.ServiceDirectory}}/${{artifact.name}}"
+          echo "##vso[task.setvariable variable=artifactSafeName]${{artifact.safeName}}"
+          if ('${{parameters.ServiceDirectory}}' -eq '*') {
+            echo "##vso[task.setvariable variable=packageLocation]core/${{artifact.name}}"
+          }
+          if ('${{parameters.ServiceDirectory}}' -eq 'cognitiveservices') {
+            echo "##vso[task.setvariable variable=packageLocation]cognitiveservices/${{artifact.directoryName}}"
+            echo "##vso[task.setvariable variable=artifactSafeName]${{artifact.directoryName}}"
+          }
+          if ('${{parameters.LibType}}' -eq 'Management') {
+            echo "##vso[task.setvariable variable=packageLocation]${{artifact.serviceDir}}/${{artifact.name}}"
+            echo "##vso[task.setvariable variable=artifactSafeName]${{artifact.name}}"
+          }
+        displayName: Set Variables for current Loop
+      - pwsh: |
+          mkdir $(artifactSafeName)/dll-docs/my-api
+          mkdir $(artifactSafeName)/dll-docs/dependencies/my-api
+          mkdir $(artifactSafeName)/dll-xml-output
+          mkdir $(artifactSafeName)/dll-yaml-output
+          mkdir $(artifactSafeName)/docfx-output
+        displayName: Create Directories for Doc Generation
+        workingDirectory: $(Build.BinariesDirectory)
+      - pwsh: dotnet build eng/service.proj /p:ServiceDirectory=$(packageLocation) /p:IncludeTests=false /p:IncludeSamples=false /p:OutputPath=$(Build.BinariesDirectory)/$(artifactSafeName)/dll-docs/my-api /p:TargetFramework=netstandard2.0
+        displayName: Build Client Packages
+        condition: and(succeeded(), eq(False, '${{ parameters.LibType }}'))
```

Comment:
Does this condition work? I don't see you setting LibType to false anywhere.

Reply:
`parameters.LibType` will evaluate as False when its not present. That's for all the client libraries